isolated-contexts.bs

```diff
@@ -25,6 +25,7 @@ spec:fetch; type:dfn; text:fetch params
 spec:fetch; type:dfn; for:fetch params; text:request
 spec:fetch; type:dfn; text:main fetch
 spec:html; type:dfn; for:environment settings object; text:cross-origin isolated capability
+spec:html; type:dfn; for:environment settings object; text:global object
```

Comment:
Is this used?

Reply:
I thought it was needed by [=environment settings object/global object=] on line 387, but it builds fine without it.